src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -267,8 +298,7 @@ public BitbucketRepository getRepository() throws IOException, InterruptedExcept
     public void postCommitComment(@NonNull String hash, @NonNull String comment) throws IOException, InterruptedException {
         String path = V2_API_BASE_URL + this.owner + "/" + this.repositoryName + "/commit/" + hash + "/build";
         try {
-            NameValuePair content = new NameValuePair("content", comment);
-            postRequest(path, new NameValuePair[]{ content });
+            postRequest(path, Arrays.asList(new BasicNameValuePair("content", comment)));
```

Comment:
You can use Collections.singletonList instead and avoid importing Arrays

Reply:
done